README.md

```diff
@@ -27,33 +29,35 @@ Goals include:
 * ***Developer friendliness***. It should be easy to customize xi editor, whether
   by adding plug-ins or hacking on the core.
 
-Please refer to the [November 2017 roadmap](https://github.com/google/xi-editor/issues/437)
-to learn more about planned features.
-
 Screenshot (will need to be updated as syntax coloring and UI polish is added):
 
-![xi screenshot](/docs/docs/img/xi-mac-screenshot.png?raw=true)
+![xi-mac screenshot](/docs/docs/img/xi-mac-screenshot.png?raw=true)
+
 
 ## Getting started
 
 This repository is the core only. You'll also need a front-end, from the list
 below.
 
+
 ### Building the core
 
-Xi targets 'recent stable Rust'. We recommend installing via [rustup](https://www.rustup.rs).
+Xi-editor targets 'recent stable Rust'. We recommend installing via [rustup](https://www.rustup.rs).
 The current minimum supported version is 1.28.
 
-To build the xi editor from the root directory of this repo:
+To build the xi-editor core from the root directory of this repo:
 
 ```
 > cd rust
 > cargo build
 ```
 
-Here are some other front-ends in various stages of development:
 
-* [xi-mac](https://github.com/google/xi-mac), the official macOS front-end.
+## Frontends
+
+Here are some front-ends in various stages of development:
+
+* [xi-mac](https://github.com/xi-editor/xi-mac), the official macOS front-end.
```

Comment:
one note: while you're in here, maybe remove the line 'and might work on other Flutter targets', below? The flutter code is now pretty fuchsia specific.